thirdparty/github.go

```diff
@@ -1358,8 +1359,9 @@ func authorizedForOrg(ctx context.Context, token, requiredOrganization, name str
 	}
 	githubClient := getGithubClient(token, caller, retryConfig{retry: true})
 
-	const numInstallationsLimit = 2000
-	opts := &github.ListOptions{PerPage: numInstallationsLimit}
+	// GitHub often appends [bot] to GitHub App usage, but this doesn't match the App slug, so we should check without this.
+	appName := strings.TrimSuffix(name, botSuffix)
```

Comment:
I'm not sure the GitHub behavior is with the names but would it make sense to check for the app name with and without the suffix?

Reply:
Honestly yeah maybe we might as well